src/components/ScreenStart.vue

```diff
@@ -94,7 +98,12 @@ export default defineComponent({
         StepperActions
     },
     setup () {
+        onUpdated(() => {
+            const { setStart } = useStepperErrors()
+            setStart(document.getElementsByClassName('has-error').length > 0)
```

Comment:
```suggestion
            setStart(titleErrors.length + messageErrors.length > 0)
```

Reply:
I experimented a bit with this, couldnt not make it work. I decided to just use the `getElementsByClassName` method for the time being. We could revisit this in the future when we have more time.

Reply:
I guess you will have to manually aggregate these errors. For instance, for the `AuthorsScreen` you will have to check to use the `authorsErrors` computed there but also check for the `orcidQueries` and `emailQueries` of every author.
In other words, this was done implicitly in the Stepper through `authors-errors.ts` but now it is moved to the screens.

One other option would be to do a similar task for the individual authors, i.e., set a state from inside AuthorCardX for each author. I feel like that is worst.

Reply:
Agreed

Reply:
Spawned #544